docs/data_access.rst

```diff
@@ -120,12 +133,36 @@ We also provide access to a ``nightly`` development build of the data, which is
 most weekday mornings. These builds are useful for beta testing new outputs, but are
 ephemeral and may not be as well validated as the ``stable`` releases.
 
+.. _access-viewer:
+
+---------------------------------------------------------------------------------------
+PUDL Viewer
+---------------------------------------------------------------------------------------
+
+We recently released the `PUDL Viewer <https://viewer.catalyst.coop/>`__ in beta.
+
+It provides flexible search of table descriptions, live data preview, and CSV
```

Comment:
not just descriptions, right? Also table names and columns? Could either add all those in or make it more generic.

Reply:
Would also be nice to mention filtering capabilities and download of the filtered data as CSV or parquet.